SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/EditorWorkerController.h

```diff
@@ -17,6 +19,14 @@ struct EditorWorkerController
 		FEditorDelegates::PrePIEEnded.Remove(PIEEndHandle);
 	}
 
+	void OnSpatialShutdown()
+	{
+		LastPIEEndTime = -1;
```

Comment:
This maybe should have gone in the last PR, but can you comment about what -1 means?